vendor.conf

```diff
@@ -47,7 +47,7 @@ github.com/xeipuuv/gojsonschema 93e72a773fade158921402d6a24c819b48aba29d
 golang.org/x/crypto 558b6879de74bc843225cde5686419267ff707ca
 golang.org/x/net 7dcfb8076726a3fdd9353b6b8a1f1b6be6811bd6
 golang.org/x/sync 450f422ab23cf9881c94e2db30cac0eb1b7cf80c
-golang.org/x/sys 07c182904dbd53199946ba614a412c61d3c548f5
+golang.org/x/sys 8dbc5d05d6edcc104950cc299a1ce6641235bc86 
```

Comment:
Perhaps this should use the same version as is used in moby/moby (95c6576299259db960f6c5b9b69ea52422860fce)

Reply:
Did a quick check, and the following packages were updated in moby (so possibly should be updated here as well)

- https://github.com/golang/sys/compare/07c182904dbd53199946ba614a412c61d3c548f5...95c6576299259db960f6c5b9b69ea52422860fce
- https://github.com/containerd/continuity/compare/22694c680ee48fb8f50015b44618517e2bde77e8...35d55c5e8dd23b32037d56cf97174aff3efdfa83
- https://github.com/docker/swarmkit/compare/872861d2ae46958af7ead1d5fffb092c73afbaf0...28f91d87bd3f75fd039dbb9be49bfd2381019261